src/test/rgw/rgw_multi/tests_es.py

```diff
@@ -8,6 +8,7 @@
 import dateutil
 
 from nose.tools import eq_ as eq
+from six.moves import range
```

Comment:
This is really not necessary, standard 'range' should work fine, since there is no a point of super optimizations, thus we don't need iterators for python2.

Reply:
@kshtsk i am not familiar with rgw tests, if you are sure about the scale of the related variable in these tests, we can surely use `range()`.

Reply:
also, i want to be clear: the ultimate goal is to remove `six` and the short term goal is to migrate to python sooner. and i really want to avoid any extra efforts unless really necessary. also i don't think to use the vanilla `range` is the safest way to do the migration.